api/fbs/v1/rpc.fbs

```diff
@@ -16,10 +16,12 @@
 // under the License.
 
 include "query.fbs";
+include "write.fbs";
 
 namespace banyandb.v1;
 
-// Query is the RPC service exposed
-rpc_service Query {
-    QueryTraces(EntityCriteria):TracesResponse;
-}
\ No newline at end of file
+// Trace is the RPC service exposed
+rpc_service Trace {
+    QueryTraces(EntityCriteria): TracesResponse;
+    WriteTraces(WriteEntity): WriteResponse(streaming: "bidi");
```

Comment:
@hanahmily Do you need `bidi`? It is typically not useful unless OAP and BanyanDB should communicate during writing data(trace).

Reply:
I would like to introduce the `reactive` pattern to provides some details about writing status. 
 1. failure counts.
 2. lastest write timestamp.
The above metrics can be exported to OAP self-ob component.

Reply:
OK, make sense.